### DISCUSSION TITLE
如何配置任务的启动条件

### DISCUSSION THREAD

**Discussion by 403728815** [apache/dolphinscheduler#15082] [Q&A]
我使用的是320版本，我需要在任务启动之前，通过sql的方式判断源数据是否准备完毕后，才启动我的任务，如果没有准备完毕，则一直循环判断，请问目前的逻辑节点如何去实现？
例如，通过count的方式，如果count（1） 大于0，则启动任务

**Comment by 403728815** [comment]
难道没人遇过类似的问题？

**Reply by heichong** [reply]
可以通过【参数传递+switch】来判定后续任务是否执行。没办法循环判定，只能把定时任务的频率调高一些
